content/influxdb/v2.0/query-data/execute-queries/flux-repl.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Query in the Flux REPL
+weight: 203
+menu:
+  influxdb_2_0:
+    name: Query in the Flux REPL
+    parent: Execute queries
+influxdb/v2.0/tags: [query]
+---
+
+The [Flux REPL](/influxdb/v2.0/tools/repl/) starts an interactive
+Read-Eval-Print Loop (REPL) where you can write and execute Flux queries.
+
+```sh
+./flux repl
+```
```

Comment:
I expect some further details in this doc, but I think that expectation can be met with a link to the REPL doc _after_ the code block. What about this:

~~~md
The **Flux REPL** starts an interactive Read-Eval-Print Loop (REPL) where you can write and execute Flux queries.

```sh
./flux repl
```

For more information, see [Use the Flux REPL](/influxdb/v2.0/tools/repl/).
~~~

Reply:
Notice you dropped "Interactive" here, shall we drop from the heading?

Reply:
Oh, that wasn't intentional 😄 . You can leave it in.